src/docs/deployment/ios.md

```diff
@@ -268,20 +268,20 @@ detailed overview of the process of releasing an app to the App Store.
 [appreview]: https://developer.apple.com/app-store/review/
 [appsigning]: https://help.apple.com/xcode/mac/current/#/dev154b28f09
 [appstore]: https://developer.apple.com/app-store/submissions/
-[appstoreconnect]: https://developer.apple.com/support/app-store-connect/
 [appstoreconnect_guide_register]: https://help.apple.com/app-store-connect/#/dev2cd126805
-[appstoreconnect_login]: https://appstoreconnect.apple.com/
 [appstoreconnect_guide]: https://developer.apple.com/support/app-store-connect/
+[appstoreconnect_login]: https://appstoreconnect.apple.com/
+[appstoreconnect]: https://developer.apple.com/support/app-store-connect/
 [codesigning_guide]: https://developer.apple.com/library/content/documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html
 [Core Foundation Keys]: https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 [devportal_appids]: https://developer.apple.com/account/ios/identifier/bundle
-[devprogram]: https://developer.apple.com/programs/
 [devprogram_membership]: https://developer.apple.com/support/compare-memberships/
-[distributionguide]: https://help.apple.com/xcode/mac/current/#/dev8b4250b57
+[devprogram]: https://developer.apple.com/programs/
```

Comment:
I'm confused by your reordering.. devprogram comes before distributionguide...

Reply:
It does! Did you mean to say that `devprogram` should come before `devprogram_membership` ? I'm making that change based on your comment above (ordering such cases by variable / line length is an old tic, from a different approach to this in case you were wondering :) )